### DISCUSSION TITLE
❤❤青衣，青衣，青衣，青衣❤❤

### DISCUSSION THREAD

**Discussion by uidsd604** [share121/inter-knot#1032] [其他]
好像让青衣狠狠的踩我，然后对我说杂鱼，杂鱼，用他那小小的，香香的玉足狠狠踩我的脸，等她最为得意的时候，偷偷拿出精心准备的粉色emp对着青衣前辈来几下，好想看到青衣前辈不敢置信的样子，等她没有力气了再捆起来，然后听着前辈的咒骂声狠狠调教，明明很厌恶我，缺又没有实力抵抗，不得不臣服我，让前辈雌堕，啊啊啊，想想就要兴奋起来了捏

**Comment by ling188** [comment]
嘿嘿青衣前辈

**Comment by ZhuYuan-148** [comment]
前辈…❤嘿嘿嘿

**Comment by SSTV-10068** [comment]
朱鸢前辈请注意形象！！！

**Comment by justafish8787** [comment]
@share121 有奇怪的东西出现了...
